### PR TITLE
add example with Selenium test using Java and Junit

### PR DESCRIPTION
**Pull request by maciejplonski-saucelabs**

### CODE REVIEW

**Review by alexplischke — COMMENTED** [review]
LGTM overall. Would be awesome if you could update the main README as well. It became a bit outdated and was/is already missing a reference to TestCafe. And now the upcoming Java-Junit-Selenium example ✨

**Comment by maciejplonski-saucelabs** [comment]
@alexplischke we plan to add more examples and to updated all the READMEs to a unified version once we're done with all of them. Does it work for you this way?

File: java-junit-selenium/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM maven:3-jdk-11-slim
```
Comment (adrian256) [inline java-junit-selenium/Dockerfile:1]: I think it would be good to use sth newer

File: java-junit-selenium/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+ENV LANG=en_US.UTF-8
+
+WORKDIR /usr/build
+COPY    . .
```
Comment (adrian256) [inline java-junit-selenium/Dockerfile:6]: shouldn't you only copy best-practice & pom.xml?

Comment (maciejplonski-saucelabs) [reply java-junit-selenium/Dockerfile]: changed

File: java-junit-selenium/Dockerfile (hunk 1 above)
Comment (adrian256) [reply java-junit-selenium/Dockerfile]: @antknight do you have any recommendations for the base image we should use in this example?

Comment (maciejplonski-saucelabs) [reply java-junit-selenium/Dockerfile]: FYI I switched to maven:3.9.6-sapmachine-11, but let's double check which one should be the way to go.

Comment (antknight) [reply java-junit-selenium/Dockerfile]: I would recommend the `3.9.6-eclipse-temurin-11` as it has less opinion on a cloud provider and just a bit smaller.  I don't have opinion on the AWS/SAP/IBM dependencies.

**Review by antknight — APPROVED** [review]
LGTM!

**Review by adrian256 — APPROVED** [review]

**Pull request merged**